doc-content/enterprise-only/admin-and-config/managing-business-central-adding-data-sources-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='managing-business-central-adding-data-sources-proc']
+= Adding a data source to {CENTRAL}
```

Comment:
Yes, @jhrcek, though its not intentional, the common steps/info will have the same language. This helps maintain consistency. But if you notice the UI element names (fields) are different in the steps. Essentially, in say, adding or editing a data source (or db drivers) some steps are going to be common/same. I havn't duplicated  the content but it may appear so.